classes/widgets.php

```diff
@@ -284,29 +284,29 @@ public function form( $instance ) {
 				<input type="checkbox" class="checkbox" id="<?php echo $this->get_field_id( $show_field ); ?>"
 				       name="<?php echo $this->get_field_name( $show_field ); ?>"<?php checked( $show ); ?> />
 				<label
-					for="<?php echo $this->get_field_id( $show_field ); ?>"><?php _e( 'Show ' . $tax_info['plural'], 'simplemap' ); ?></label>
+					for="<?php echo $this->get_field_id( $show_field ); ?>"><?php _e( 'Show ' . $tax_info['plural'], 'SimpleMap' ); ?></label>
 				<br/>
 
 				<?php
 				/** TODO: The commented out code below isn't working yet. Implement it.
 				 * $values = isset( $instance[$key] ) ? esc_attr( $instance[$key] ) : '';
 				 * ?>
-				 * <p><label for="<?php echo $this->get_field_id( $key ); ?>"><?php _e( $tax_info['plural'] . ':', 'simplemap' ); ?></label>
+				 * <p><label for="<?php echo $this->get_field_id( $key ); ?>"><?php _e( $tax_info['plural'] . ':', 'SimpleMap' ); ?></label>
 				 * <input class="widefat" id="<?php echo $this->get_field_id( $key ); ?>" name="<?php echo $this->get_field_name( $key ); ?>" type="text" value="<?php echo $values; ?>" /></p>
 				 * <?php
 				 */
 			}
 
 			/** TODO: The commented out code below isn't working yet. Implement it.
-			 * <p><label for="<?php echo $this->get_field_id( 'default_lat' ); ?>"><?php _e( 'Default Lat:', 'simplemap' ); ?></label>
+			 * <p><label for="<?php echo $this->get_field_id( 'default_lat' ); ?>"><?php _e( 'Default Lat:', 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Line exceeds 120 characters; contains 134 characters

<!-- Comment by [SideCI](https://sideci.com) -->